committers.md

```diff
@@ -86,3 +86,45 @@ non-voting project maintainers.
     {% endif %}
   {% endfor %}
 </tbody></table>
+
+### **Becoming a committer**
+
+There are many ways to [contribute](https://arrow.apache.org/docs/developers/contributing.html)
+to the Apache Arrow project, including issue reports,
+documentation, tests, and code. Contributors with sustained, high-quality activity
+may be invited to become a committer by the PMC.
+
+Becoming a committer is a recognition of sustained
+contribution to the project, and comes with the privilege of
+committing changes directly in all Arrow github repositories. Becoming
+a committer is also a significant responsibility, and committers are
+expected to use their status and access to improve the Arrow project
+for the entire community.
+
+When considering to invite someone to be a committer, the PMC looks for
+contributors who are doing the work and exercising the judgment expected
+of a committer already. After all, any contributor can do all of the things a
+committer does except for merge a PR. While there is no set list of
+requirements, nor a checklist that entitles one to commit privileges,
+typical behaviors include:
+
+* Contributions beyond pull requests, such as reviewing other pull requests,
+  fixing bugs and documentation, triaging issues, answering community
+  questions, improving usability, reducing technical debt, helping
+  with CI, verifying releases, debugging in strange environments, etc.
+
+* Consistent, significant, and sustained contributions in the Arrow
+  community, typically on the order of 6 months or more.
+
+* Assistance growing the size and health of the community via constructive and
```

Comment:
Good opportunity to link to the code of conduct, and maybe there's a way to link to some ASF reference on The Apache Way or something.

Reply:
Great idea -- added in 004f8b77931